config.yaml

```diff
@@ -22,6 +20,9 @@ default:
  commit_updates: True
  push_updates: True
  force_rebuild: False
+release_notes:
+ release_notes_repo_url: "https://github.com/open-power-host-os/open-power-host-os.github.io.git"
+ release_notes_repo_branch: "master"
```

Comment:
@alanoe is making all options here match the same name as in `config.py`. Having a `repo_url` command line option there would be ambiguous. I believe this can be improved in future PRs, when options will be properly separated among commands.